### PR TITLE
PSMDB Telemetry Changes Phase-1

### PR DESCRIPTION
**Pull request by surbhat1595**

### CODE REVIEW

File: percona-packaging/redhat/percona-server-mongodb.spec.template [hunk 1]
```diff
@@ -394,10 +397,24 @@ fi
...
+chgrp percona-telemetry /usr/local/percona/telemetry_uuid
+chmod 664 /usr/local/percona/telemetry_uuid
 rm -f /tmp/call-home.sh
 
+%post mongos
+mkdir -p /usr/local/percona/telemetry/psmdbs
+chown mongod:percona-telemetry /usr/local/percona/telemetry/psmdbs
+chmod 775 /usr/local/percona/telemetry/psmdbs
+chmod g+s /usr/local/percona/telemetry/psmdbs
+chmod u+s /usr/local/percona/telemetry/psmdbs
```
Comment (EvgeniyPatlan) [inline percona-packaging/redhat/percona-server-mongodb.spec.template:416]: install -d -m 2775 -o mongod -g percona-telemetry /usr/local/percona/telemetry/psmdbs

File: percona-packaging/redhat/percona-server-mongodb.spec.template [hunk 2]
```diff
@@ -394,10 +397,24 @@ fi
 %endif
     fi
 fi
+mkdir -p %{psmdb_telemetry}
```
Comment (EvgeniyPatlan) [inline percona-packaging/redhat/percona-server-mongodb.spec.template:400]: install -d -m 2775 -o mongod -g percona-telemetry /usr/local/percona/telemetry/psmdbs

File: percona-packaging/debian/percona-server-mongodb-server.postinst [hunk 3]
```diff
@@ -114,4 +114,12 @@ else
     fi
 fi
 
+if [ -d /usr/local/percona/telemetry ]; then
+    mkdir -p /usr/local/percona/telemetry/psmdb
+    chown mongod:percona-telemetry /usr/local/percona/telemetry/psmdb
+    chmod 775 /usr/local/percona/telemetry/psmdb
+    chmod g+s /usr/local/percona/telemetry/psmdb
+    chmod u+s /usr/local/percona/telemetry/psmdb
```
Comment (EvgeniyPatlan) [inline percona-packaging/debian/percona-server-mongodb-server.postinst:122]: install -d -m 2775 -o mongod -g percona-telemetry /usr/local/percona/telemetry/psmdbs

File: percona-packaging/debian/percona-server-mongodb-mongos.postinst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+if [ -f /usr/share/debconf/confmodule ]; then
+  . /usr/share/debconf/confmodule
+fi
+
+if [ -d /usr/local/percona/telemetry ]; then
+    mkdir -p /usr/local/percona/telemetry/psmdbs
+    chown mongod:percona-telemetry /usr/local/percona/telemetry/psmdbs
+    chmod 775 /usr/local/percona/telemetry/psmdbs
+    chmod g+s /usr/local/percona/telemetry/psmdbs
+    chmod u+s /usr/local/percona/telemetry/psmdbs
```
Comment (EvgeniyPatlan) [inline percona-packaging/debian/percona-server-mongodb-mongos.postinst:18]: install -d -m 2775 -o mongod -g percona-telemetry /usr/local/percona/telemetry/psmdbs

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: percona-packaging/redhat/percona-server-mongodb.spec.template (hunk 1 above)
Comment (surbhat1595) [reply percona-packaging/redhat/percona-server-mongodb.spec.template]: Good point, Thank you.

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by igorsol — COMMENTED** [review]
LGTM

**Pull request merged**